packages/terra-tabs/src/common-tabs/_Tabs.jsx

```diff
@@ -59,22 +67,52 @@ const propTypes = {
    */
   onChange: PropTypes.func,
   /**
-   * Whether or not the tab is draggable.
+   * Callback function when add button selection has changed.
+   * Parameters: 1. Event 2. Selected pane's key
    */
+  onSelectAddButton: PropTypes.func,
+  /**
+   * Callback function when a tab is closing.
+   * It receives two parameters: 1. Item key of the tab to close. 2. MetaData associated with the tab.
+   */
+  onClosingTab: PropTypes.func,
+
+  /**
+   * Indicates whether the tabs are closable, allowing the user to close tabs.
+   */
+  isClosable: PropTypes.bool.isRequired,
```

Comment:
This should not be a required prop

Reply:
Done , thanks.  https://github.com/cerner/terra-framework/commit/f4bd3b60b2862061fc633330bda690d094cd0ea6